fele-client-service/restful-api/routes/fele-client.route.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const router = express.Router()
+
+const { createNetworkHandler,
+    deleteNetworkHandler,
+    removeOrganizationHandler,
+    createChannelHandler,
+    addOrganizationHandler,
+    chainCodeDeployHandler } = require('../handlers/fele-client.handler')
+
+router.get('/', async (req, res) => {
+    res.send("Im FELE-CLIENT Base path")
+})
+
+router.post('/createNetwork', createNetworkHandler)
+router.post('/createChannel', createChannelHandler)
+router.delete('/deleteNetwork', deleteNetworkHandler) 
+router.put("/network/addOrg", addOrganizationHandler) //TODO
+router.put("/network/remove/:orgName", removeOrganizationHandler) //TODO
```

Comment:
delete method ?

Reply:
HTTP DELETE to delete the network. We can delete the network right?